recipes/or-tools/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "or-tools" %}
+{% set version = "9.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/google/or-tools/archive/v{{ version }}.tar.gz
+  sha256: fa7700b614ea2a5b2b6e37b76874bd2c3f04a80f03cbbf7871a2d2d5cd3a6091
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - git
+    - pkg-config
+  host:
+    - abseil-cpp 20210324.1
+    - gflags
+    - glog
```

Comment:
we don't depends on gflags and glog since v8.0 IIRC

Reply:
Thanks, removed.